mainapp/views.py

```diff
@@ -31,25 +28,47 @@ def create(request):
             prod.is_active = 1
             prod.author = Profile.objects.get(user=request.user)
             prod.save()
-        return render(request, 'mainapp/home.html'  )
+        return render(request, 'mainapp/home.html')
     else:
         form = ProductForm()
         return render(request, 'mainapp/create.html', {'form': form})
 
 
 def comment(request):
-    com = Comment.objects.all()
-    return render(request, 'mainapp/comment.html', {'title': 'Коментарccc', 'comm': com})
+    prod = request.GET.get('')
+    if request.method == 'POST':
```

Comment:
`request.method == 'POST' and request.user.is_authenticated`

Reply:
changed